include/roboteam_utils/LineSegment.h

```diff
@@ -122,6 +126,21 @@ class LineSegment {
      */
     [[nodiscard]] bool operator==(const LineSegment &other) const;
 
+    /**
+     * @return Returns the vector which represents the direction in which the line points, e.g. end-start
+     */
+    [[nodiscard]] Vector2 direction() const;
+
+
+    /**
+      * Moves the line's start and end points by the given vector
+      */
+    void move(const Vector2& by);
+
+
+    [[nodiscard]] std::optional<Vector2> firstIntersects(const LineSegment &segment) const;
```

Comment:
Documentation would be nice. It is not immediately obvious what this function does.